core/src/main/java/feast/core/model/FeatureTable.java

```diff
@@ -359,13 +359,13 @@ public void delete() {
 
   public String protoHash() {
     List<String> sortedEntities =
-        this.getEntities().stream().map(entity -> entity.getName()).collect(Collectors.toList());
-    Collections.sort(sortedEntities);
+        this.getEntities().stream().map(EntityV2::getName).sorted().collect(Collectors.toList());
 
-    List<FeatureV2> sortedFeatures = new ArrayList(this.getFeatures());
     List<FeatureSpecV2> sortedFeatureSpecs =
-        sortedFeatures.stream().map(featureV2 -> featureV2.toProto()).collect(Collectors.toList());
-    sortedFeatures.sort(Comparator.comparing(FeatureV2::getName));
```

Comment:
The original code sort `sortedFeatures` after they have been used for generating `sortedFeatureSpecs`, which I don't get why.